build.sbt

```diff
@@ -26,36 +26,49 @@ libraryDependencies ++= Seq(
     "com.tinkerpop.gremlin" % "gremlin-groovy" % "2.5.0",
     "com.tinkerpop.gremlin" % "gremlin-java" % "2.5.0",
     "org.specs2" %% "specs2" % "2.3.13" % "test",
-    "com.thinkaurelius.titan" % "titan-core" % "0.5.0" % "provided",
-    "com.thinkaurelius.titan" % "titan-berkeleyje" % "0.5.0" % "test"
+    "com.thinkaurelius.titan" % "titan-core" % "0.5.2" % "provided",
+    "com.thinkaurelius.titan" % "titan-berkeleyje" % "0.5.2" % "test"
     )
 
+
+scalacOptions ++= {
+  CrossVersion.partialVersion(scalaVersion.value) match {
+    // if scala 2.11+ is used, be strict about compiler warnings
+    case Some((2, scalaMajor)) if scalaMajor >= 11 =>
+      Seq("-Xfatal-warnings")
+    case _ =>
+      Nil
+  }
+}
+
+
+
 sonatypeSettings
 
 profileName := "com.ansvia"
 
 publishTo <<= version { (v:String) =>
-      val nexus = "https://oss.sonatype.org"
-      if(v.trim.endsWith("SNAPSHOT") || """.+\-\d{8}+$""".r.pattern.matcher(v.trim).matches())
-          Some("snapshots" at nexus + "/content/repositories/snapshots")
-      else
-	  Some("releases"  at nexus + "/service/local/staging/deploy/maven2")
-  }
+  val nexus = "https://oss.sonatype.org"
```

Comment:
I'm using 4 spaces as standard indentation, it is okay if I convert all 2 spaces into 4 spaces indentation?